CONTRIBUTING.md

```diff
@@ -0,0 +1,248 @@
+# Contributing to Lux
+
+Thank you for your interest in contributing to Lux! This document will help
+answer any outstanding questions you may have about the contribution process.
+
+*Please read the [Code of Conduct](./CODE_OF_CONDUCT.md) before you participate
+in community.*
+
+## Contents
+
```

Comment:
Suggestion: include a definition of what we consider a contribution to Lux, a _broad_ definition. Code for sure--and most of this document will address that type of contribution--but also: updating and writing documentation, answering questions and chatting with the community in the Gitter room, filing, organizing, commenting on issues, teaching others how to use Lux, making suggestions on branding, building community, outreach, etc. I think one of the most common mistakes OSS projects make is focusing on code when so much more goes into writing software as a community.